cachito/workers/tasks/pip.py

```diff
@@ -8,6 +11,9 @@
 from cachito.workers.tasks.celery import app
 
 
+log = logging.getLogger(__name__)
```

Comment:
I'm not using it for my code, but I'll just leave it here because I'm sure we'll be using it in the other parts of the file.